packages/lib/src/vis/line/LineVis.tsx

```diff
@@ -152,14 +153,33 @@ function LineVis(props: Props) {
                 {`${abscissaLabel ?? 'x'} = ${formatTooltipVal(abscissa)}`}
 
                 <div className={styles.tooltipValue}>
-                  <strong>{formatTooltipVal(value)}</strong>
-                  {error && ` ±${formatTooltipErr(error)}`}
-                  {dtype && <em>{` (${formatNumType(dtype)})`}</em>}
+                  {auxiliaries.length > 0 && (
+                    <span
+                      className={styles.mark}
+                      style={{ color: curveColor }}
+                    />
+                  )}
+                  <span>
+                    <strong>{formatTooltipVal(value)}</strong>
+                    {error && ` ±${formatTooltipErr(error)}`}
+                    {dtype && <em>{` (${formatNumType(dtype)})`}</em>}
+                  </span>
                 </div>
 
-                {auxiliaries.map(({ label, array }) => {
+                {auxiliaries.map(({ label, array }, index) => {
                   const val = formatTooltipVal(array.get(xi));
-                  return <div key={label}>{`${label} = ${val}`}</div>;
+                  const color = auxColors[index] || lastAuxColor;
+
+                  return (
+                    <div
+                      className={styles.tooltipAux}
+                      key={label}
+                      data-color={color}
```

Comment:
Why do you need this attribute ?

Reply:
I don't, thanks. Left-over from another attempt of using `background-color: attr(data-color)` in CSS, but this doesn't currently work across browsers -- `attr()` only works with the `content` CSS property. https://devdocs.io/css/attr()